e2e/tests/functional/plugins/gauge/gauge.e2e.spec.js

```diff
@@ -133,4 +133,50 @@ test.describe('Gauge', () => {
 
     // TODO: Verify changes in the UI
   });
+
+  test('Gauge does not display NaN when data not available', async ({ page }) => {
+    // Create a Gauge
+    await createDomainObjectWithDefaults(page, {
+      type: 'Gauge',
+      name: 'Unnamed Gauge'
+    });
+
+    // Create a Sine Wave Generator in the Gauge with a loading delay
+    await page.getByRole('button', { name: ' Create ' }).click();
+    await page.getByRole('menuitem', { name: ' Sine Wave Generator' }).click();
+    await page
+      .getByRole('dialog')
+      .locator('div')
+      .filter({ hasText: 'Loading Delay (ms)' })
+      .nth(3)
+      .click();
+    await page.getByLabel('Loading Delay (ms)').fill('5000');
+    await page.locator('[aria-label="Save"]').click();
+    await page.getByLabel('Expand My Items folder').click();
+    await page
+      .getByRole('treeitem', { name: 'Expand Unnamed Gauge gauge  Unnamed Gauge' })
+      .locator('a')
+      .click();
+    const gaugeNoDataText = await page.locator('.js-dial-current-value tspan').textContent();
```

Comment:
This doesn't need to block this PR from going in, but we should consider adding better ARIA to Gauges to avoid plain CSS selectors.

Reply:
Agree, but I'd like to add aria's across all the gauge display styles in a future issue.